elixir/apps/domain/lib/domain/policies/policy/changeset.ex

```diff
@@ -0,0 +1,58 @@
+defmodule Domain.Policies.Policy.Changeset do
+  use Domain, :changeset
+  alias Domain.Auth
+  alias Domain.Policies.Policy
+
+  @fields ~w[name actor_group_id resource_id]a
+  @update_fields ~w[name]a
+  @required_fields @fields
+
+  def create_changeset(attrs, %Auth.Subject{} = subject) do
+    %Policy{}
+    |> cast(attrs, @fields)
+    |> validate_required(@required_fields)
+    |> changeset()
+    |> put_change(:account_id, subject.account.id)
+    |> put_change(:created_by, :identity)
+    |> put_change(:created_by_identity_id, subject.identity.id)
+    |> unique_constraint([:account_id, :name],
```

Comment:
This should be moved to `changeset/1` because you can change name in `update` and get a conflict there too. The same goes to all other `*_constraint`'s, it doesn't hurt to reuse them even if we don't expect them to trigger.

Reply:
Sounds good 👍

Reply:
This should be updated